src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const galleryList = document.querySelector('ul');
+const mainImage = document.querySelector('.gallery__large-img');
+
+galleryList.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const link = e.target.closest('.list-item__link');
+
+  mainImage.src = link.href;
```

Comment:
There is a potential issue here if the `e.target` is not within an element with the class `list-item__link`. In such cases, `link` will be `null`, and attempting to access `link.href` will result in a runtime error. Consider adding a check to ensure `link` is not null before accessing its `href` property.